Products/PleiadesEntity/profiles/default/types/Connection.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<object name="Connection"
+        meta_type="Factory-based Type Information with dynamic views"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="plone">
+   
+ <property name="title" i18n:translate="">Connection</property>
+ <property name="description" i18n:translate="">Establishes a connection to another place.</property>
+ <property name="content_icon">map_icons/connection-green.png</property>
```

Comment:
Does this load the icon relative to the current context, or relative to the portal? If the former, better to use `icon_expr` and build an absolute url